compare_ll.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+
+# In-package imports
+import corpus_utils
+import bucketers
+
+def print_header(header):
```

Comment:
Since these two functions are used in both `compare_mt.py` and `compare_ll.py`, maybe we can move them to a different file (`arg_utils.py`?)